File: C++/OOP-Practice/level1.cpp
#include <string>
#include <iostream>

class Car{

    public:
        std::string model;
        int year;
        std::string version;

        Car(std::string model, int year, std::string version){
            this->model = model;
            this->year = year;
            this->version = version;
        }
};

int main() {
Car car("F-150", 2025, "Tremor");
std::cout << car.model << " " << car.year << " " << car.version << std::endl;
}

File: Java/OOP-Practice/level1.java
class Car {
    
    //atributos

    String model;
    String version;
    int year;
    
    //constructors
   Car(String model, int year, String version){
        this.model = model;
        this.year = year;
        this.version = version;
    }
    //métodos
    

    //main
    public static void main(String[] args){
        Car car = new Car("F-150", 2025, "Tremor");
        System.out.println(car.model + " " + car.year + " " + car.version);

    }

    
}